### PR TITLE
Trlst 589 revise setup

### PR DESCRIPTION
**Pull request by holdenweb**
Make the build a easier for developers new to the project.

This PR changes the README and Makefile to simplify the establishment of
a local docker-based development setup.

Chris Pettinga plans to improve testing also.

### CODE REVIEW

**Review by holdenweb — COMMENTED** [review]
Personally I'd have made SSH the default, because this is a developer setup and I'm more used to authenticating with SSH. Since the instructions are now explicit, however, this is quite OK.

**Review by KnowYourLines — APPROVED** [review]
Works for me

File: Makefile [hunk 1]
```diff
@@ -145,20 +150,11 @@ endif
...
-
-pytest:
-ifdef service
-	docker-compose run -e $(CIRCLECI) --rm $(service) pytest --ignore=staticfiles -n 4
+	docker-compose run -w /app -e $(CIRCLECI) --rm $(service) pytest -m "not version2" --ignore=staticfiles -n 4
 else
-	docker-compose run --rm api pytest --ignore=staticfiles -n 4
-	docker-compose run --rm public pytest --ignore=staticfiles -n 4
-	docker-compose run --rm caseworker pytest --ignore=staticfiles -n 4
+	docker-compose run -w /app --rm api pytest -m "not version2" --ignore=staticfiles -n 4
```
Comment (holdenweb) [inline Makefile:155]: Note for the future: a loop such as 

    for system in api public caseworker
    do
        docker-compose run -w /app --rm $system \
            pytest -m "not version2" --ignore=staticfiles -n
    done

would explain intent equally well and underline the fact that the same command is being run on all three systems.

**Review by holdenweb — COMMENTED** [review]
I can't approve my own PR, but given that Johnny used gthis branch for a successful build I'd say we're good to go!

**Comment by holdenweb** [comment]
I can, however, merge it!

**Pull request merged**